### PR TITLE
Prepare for 1.1 release

### PR DESCRIPTION
**Pull request by CharliePoole**
After merging to master, this will become the 1.1 release of the GUI.

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -1,4 +1,41 @@
-﻿TestCentric Runner for NUnit 1.0.1 - August 15, 2019
+﻿TestCentric Runner for NUnit 1.1.0 - November 5, 2019
+
+General
+
+This release incluseds new features beyond the 1.0 release, together with some bug fixes.
```
Comment (mikkelbu) [inline CHANGES.txt]:
```suggestion
This release includes new features beyond the 1.0 release, together with some bug fixes.
```

File: CHANGES.txt [hunk 2]
```diff
@@ -1,4 +1,41 @@
...
+ * 394 Update engine
+ * 395 Display version and path of loaded extensions
+ * 397 Fix unhandled NullReferenceException thrown when test fails with Assert.Fail()
+ * 399 Show the test assembly name in the main view's title bar.
+ * 402 Set Project Configuration to use
+ * 406 FIXED: Appveyor rebuild causes error
+ * 412 Update link to Chocolatey package
+ * 414 FIXED: Testcentric.exe needs icon
+ * 415 Better installation documentation
+ * 424 Use common dialog for exensions
```
Comment (mikkelbu) [inline CHANGES.txt]:
```suggestion
 * 424 Use common dialog for extensions
```

**Comment by CharliePoole** [comment]
@mikkelbu Thanks for reviewing! Did you have a chance to try out the latest master?

**Comment by mikkelbu** [comment]
@CharliePoole I've not had time. I will try to see if I have time tomorrow, but currently I've very little sparetime.

**Pull request merged**